File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let ans = 0; 
        for(let i = 0; i< nums.length; i++)
        {
            let cnt = 0;
           
            while(Math.floor(nums[i]) > 0)
            {
                nums[i] = nums[i]/ 10; 
                cnt++;
            }
            
            if(cnt %2 ==0)
                ans++;
        }return ans;
    
};

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.cs
public class Solution {
    public int FindNumbers(int[] nums) {
        int ans = 0; 
        for(int i = 0; i< nums.Length; i++)
        {
            int cnt = 0; 
            while(nums[i] > 0)
            {
                nums[i]/=10;cnt++;
            }
            if(cnt %2 ==0)
                ans++;
        }return ans;
    }
}